### DISCUSSION TITLE
Dudas sobre Kata 10

### DISCUSSION THREAD

**Discussion by ArmandoRiveraT** [LaunchX-InnovaccionVirtual/CursoIntroPython#254] [Q&A]
Tengo dudas sobre si lo que vamos a entregar son los archivos .py o si solo serán capturas como en la kata 2? Alguien sabe?

**Comment by jstoledano** [comment]
Ya contestaron esa pregunta aquí: https://github.com/LaunchX-InnovaccionVirtual/CursoIntroPython/discussions/223 mira si te sirve la respuesta.

**Comment by doguedogue** [comment]
Puedes enviar notebook (.ipnb) y/o python (.py), toda evidencia de tus ejercicios es importante
